src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -188,6 +199,33 @@ def _get_rop_steps(rop: hou.Node):
     return rop_steps
 
 
+# Any changes to this function should be reflected in the user guide.
+def _get_render_strategy_for_node(node: hou.Node) -> RenderStrategy:
+    render_strategy = RenderStrategy.PARALLEL
+
+    if (
+        node.type().nameWithCategory() == "Driver/geometry"
+        and node.parm("initsim")
+        and node.parm("initsim").eval()
+    ):
+        render_strategy = RenderStrategy.SEQUENTIAL
+
+    if node.parm("deadline_cloud_render_strategy"):
+        strategy_string = node.parm("deadline_cloud_render_strategy").evalAsString()
+        if strategy_string.upper() == RenderStrategy.SEQUENTIAL.value:
+            render_strategy = RenderStrategy.SEQUENTIAL
+        elif strategy_string.upper() == RenderStrategy.PARALLEL.value:
+            render_strategy = RenderStrategy.PARALLEL
+        elif strategy_string == "":
+            # Use the submitter's default choice
```

Comment:
I added this in here so that someone could make a dropdown menu for override options:
<img width="283" alt="Screenshot 2024-05-22 at 7 30 55 AM" src="https://github.com/aws-deadline/deadline-cloud-for-houdini/assets/6042774/c5a63255-a085-46de-89f1-ce2b59dfaaea">

If default is a supported option, it can be added to every node and only changed for specific nodes that need overriding.

Someone that's adding a parameter as just an editable string probably wouldn't use this feature though.

Reply:
Is there a use-case for wanting to add it to all nodes? A user would have to script something to do this since they'd be editing every single node in the graph?

My understanding would be that we shouldn't provide a `default` option, because we will inevitably change what that default is for a particular node set-up and a user has no insight on what we think the default is.